packages/api/test/datasets/refresh-dataset

```diff
@@ -0,0 +1,388 @@
+#! /usr/bin/env node
```

Comment:
We use this same script and dataset in [@readme/openapi-parser](https://npm.im/@readme/openapi-parser) this one just has a lot more exclusions.

https://github.com/readmeio/openapi-parser/blob/main/bin/refresh-real-world-apis